connectivity/FEATURE_BLE/CMakeLists.txt

```diff
@@ -1,10 +1,12 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+add_library(mbed-os-ble OBJECT)
```

Comment:
I assume we should just use `mbed-` prefix as everywhere else. I can refactor  to remove it.